### ISSUE TITLE
Cannot see all labels when specifying a large amount of them

### ISSUE THREAD

**Issue by jesusdanielportillo** [Microsoft/VoTT#218]
I was using 60 labels with this tool but on screen I can see between 30 and 40 labels to use with the images.

How can I see all the labels specified while selecting an image directory?

Thank you

**Comment by lukeatagl** [comment]
Hi @jesusdanielportillo I've also experienced this issue. I used a quick hacky workaround to add a scrollbar to the window that contains these tags

1. Open the 'Developer Console' from the 'Debug' menu

![image](https://user-images.githubusercontent.com/17837086/44066884-26ebf2ba-9fb6-11e8-8d71-8d9d71c2bd43.png)

2. Click on the 'Console' tab

![image 1](https://user-images.githubusercontent.com/17837086/44066894-329ad2fc-9fb6-11e8-98ae-2bd458c4ed78.png)

3. Paste in the JavaScript snippet below, and then hit your enter key

```
document.getElementById("tagsContainer").style.overflow = "scroll";
document.getElementById("tagsContainer").style.height = "100px";
```

You should then see a vertical scrollbar on the area that contains the tags.

**Comment by wbreza** [comment]
Resolved in v2.0.0
